src/compatTest/kotlin/io/github/gradlenexus/publishplugin/NexusPublishPluginTests.kt

```diff
@@ -294,7 +294,10 @@ class NexusPublishPluginTests {
     }
 
     @Test
-    fun `publishes to two Nexus repositories`(@MethodScopeWiremockResolver.MethodScopedWiremockServer @Wiremock otherServer: WireMockServer) {
+    fun `publishes to two Nexus repositories`(
+        @MethodScopeWiremockResolver.MethodScopedWiremockServer @Wiremock
+        otherServer: WireMockServer
+    ) {
```

Comment:
Minor think, but I wonder if we are ok with single annotation formatting such as:

```    
fun `publishes to two Nexus repositories`(
        @Wiremock otherServer: WireMockServer,
        @Wiremock yetAnotherServer: WireMockServer
    ) {
```
?

And if Spotless is able to handle that?

Reply:
Not sure about that. This is my first time dealing with spotless. The aim was to get the Gradle fixes, everything else is just the consequence of fixes that brings. I think this PR should only do formatting changes.

As far as I understand your question, it's about whether the Wiremock Jupiter extension is providing a new instance or a reused instance for each parameter, and then whether ktlint can format single annotation on a single line. Not sure about either :)